codecov.yml

```diff
@@ -7,6 +7,6 @@ coverage:
     patch:
       default:
         enabled: yes
-        target: 100%
+        target: 72%
```

Comment:
Could even bump this to 74%, since that's what you reached :) https://github.com/openedx/xapi-db-load/actions/runs/5381452652/jobs/9765542187?pr=32

Reply:
Done